californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -205,10 +213,11 @@ public BlockwiseLayer(final NetworkConfig config) {
 		block1Transfers.setEvictingOnReadAccess(false);
 		block2Transfers = new LeastRecentlyUsedCache<>(maxActivePeers, TimeUnit.MILLISECONDS.toSeconds(blockTimeout));
 		block2Transfers.setEvictingOnReadAccess(false);
-
+		strictBlock2Option = config.getBoolean(NetworkConfig.Keys.BLOCKWISE_STRICT_BLOCK2_OPTION, NetworkConfigDefaults.DEFAULT_BLOCKWISE_STRICT_BLOCK2_OPTION);
+		
 		LOGGER.info(
-			"BlockwiseLayer uses MAX_MESSAGE_SIZE={}, PREFERRED_BLOCK_SIZE={}, BLOCKWISE_STATUS_LIFETIME={} and MAX_RESOURCE_BODY_SIZE={}",
-			new Object[]{maxMessageSize, preferredBlockSize, blockTimeout, maxResourceBodySize});
+			"BlockwiseLayer uses MAX_MESSAGE_SIZE={}, PREFERRED_BLOCK_SIZE={}, BLOCKWISE_STATUS_LIFETIME={}, MAX_RESOURCE_BODY_SIZE={}, BLOCKWISE_STRICT_BLOCK2_OPTION={}",
+			new Object[]{maxMessageSize, preferredBlockSize, blockTimeout, maxResourceBodySize, strictBlock2Option});
```

Comment:
In the meantime, we usually use the implicit array instead of the explicit array, what we used at the beginning of introducing slf4j. So usually we adapt that, if we're on it. But that's no big thing.

```
		LOGGER.info(
			"BlockwiseLayer uses MAX_MESSAGE_SIZE={}, PREFERRED_BLOCK_SIZE={}, BLOCKWISE_STATUS_LIFETIME={}, MAX_RESOURCE_BODY_SIZE={}, BLOCKWISE_STRICT_BLOCK2_OPTION={}",
			maxMessageSize, preferredBlockSize, blockTimeout, maxResourceBodySize, strictBlock2Option);
```

Reply:
That's fine for me, I'm also used to use the implicit array normally. I will make the change tonight and update the PR.